pages/profile/read/content/index.js

```diff
@@ -36,6 +36,7 @@ module.exports = merge({}, baseContent, {
     title: 'Personal licence',
     noPil: 'None',
     incompletePil: 'Application started.',
+    incompleteOtherEst: 'Application started at another establishment',
```

Comment:
If "Application started." has a full stop then this probably should too.
```suggestion
    incompleteOtherEst: 'Application started at another establishment.',
```